odbc/src/api/errors.rs

```diff
@@ -39,15 +46,37 @@ pub enum ODBCError {
     )]
     MissingDriverOrDSNProperty,
     #[error(
-        "[{}][API] Buffer size '{0}' not large enough for string",
+        "[{}][API] Buffer size \"{0}\" not large enough for data",
         VENDOR_IDENTIFIER
     )]
     OutStringTruncated(usize),
+    #[error(
+        "[{}][API] floating point data \"{0}\" was truncated to fixed point",
```

Comment:
I've added a similar comment somewhere in the test.
Can we try to not print out user data in any of the error message?